api/config-dev.yaml

```diff
@@ -62,6 +66,11 @@ SparkAppConfig:
   TTLSecond: 86400
 RouterDefaults:
   Image: localhost:5000/turing-router
+  # Uncomment the following for running E2E tests locally OR to use proprietary experiment engine
```

Comment:
Since we can specify multiple config files on app start up (and the later ones will override the former ones), should we break the experiment engine configs into a separate file and add instructions to include it only if intending to use the experiment engine ?